sdk/src/test/AllTests.ls

```diff
@@ -30,9 +30,11 @@ package {
      */
     public class AllTests {   
         public static function main() {
-            var result:TestResult = TestRunner.runAll((AllTests as Type).getAssembly());
-            
-            Process.exit(result.typeReport.successful ? 0 : 1);
+            TestRunner.onComplete += function(result:TestResult) {
+                
+                Process.exit(result.typeReport.successful ? 0 : 1);
+            };
+            TestRunner.runAll((AllTests as Type).getAssembly(), true);
```

Comment:
Hmm, if we're running the tests asynchronously, perhaps switching from static methods/callbacks to instance ones would make more sense e.g. create a `TestRunner`, set a callback for the delegate and call `runAll` on it. If it's static we can only run one test at a time (or rather, until the previous one completes).

That didn't matter when runAll was synchronous, because you can only run one method at a time anyway.